source/Scrapers/TeamCityQueueLengthScraper.cs

```diff
@@ -80,10 +69,8 @@ protected override void Scrape()
             {
                 // if not present, reset the gauge to zero
                 waitReasonsGauge.WithLabels(item.buildTypeId, item.waitReason).Reset();
-                consoleString.AppendLine($"{item} | 0");
+                Logger.Debug("Build Type {BuildTypeId}, Wait Reason {WaitReason}, Count {Count}", item.buildTypeId, item.waitReason, 0);
```

Comment:
This is a good translation of the old log line, however I wonder if it'd be more appropriate to say 'not observed' here, or something. It wasn't observed _at zero_, it was missing entirely. Not sure what to do.

Reply:
I'm not sure either - I'm going to stick with the one to one mapping (well, the fixed version of the old log line)